.env.example

```diff
@@ -43,6 +45,10 @@ HEIMDALL_DOCKER_BUILD_CONTEXT="https://github.com/maticnetwork/heimdall.git#deve
 VERBOSE=true# if set to true will print logs also from remote machines
 DD_API_KEY=DATADOG_API_KEY# Datadog API key
 
+# Bor and Heimdall Snapshot related variables (when running mainnet/testnet node)
+BOR_SNAP_URL= # URL for bor data snapshot (refer: https://snapshot.polygon.technology/)
```

Comment:
can we add examples here?

Reply:
Added: [88a5694](https://github.com/maticnetwork/matic-cli/pull/157/commits/88a56943bc7a38b12a738340a89a6db70e4bd69f)